### DISCUSSION TITLE
GitHub Education UI Redesign - Feedback

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#113185] [GitHub Education]
**_The purpose of this discussion is to collect feedback from our verified students on our updated GitHub Education platform User Interface (UI) design._** 

We're excited to roll out our redesigned **GitHub Education platform**. This overhaul marks a significant leap towards a more user-friendly, efficient, and engaging learning environment.

#### How to Access the GitHub Education UI Redesign:

Get [verified as a learner](https://education.github.com/discount_requests/application) on GitHub Education to gain access to the new GitHub Education platform - everything you need to pursue a career in tech.

![edu_ui_beta](https://github.com/community/community/assets/4041275/ba19202f-6a4f-4e18-8916-eca08b584583)

#### What's New:

🚀**Sleek Design**: Our new UI boasts a modern and sleek design, aimed at enhancing usability and aesthetics. Whether you're browsing GitHub Education resources, checking assignments, or looking to connect with your peers, you'll find the experience smoother than ever.

🎨 **Streamlined Navigation**: We've reimagined the layout to ensure seamless navigation, making it easier for you to access essential features and resources. Say goodbye to cluttered menus! Our new side navigation groups similar topics and experiences together, making it easier than ever to find what you need. 

📣 **Enhanced Notifications**: We have also improved our platform's notifications to keep you posted on the important things that need your attention.

<img width="1450" alt="Get_Connected_EDU" src="https://github.com/community/community/assets/4041275/18105ef4-874b-4526-8ed9-0a6b5ed9c8e2">

#### How You Can Help:

Your feedback is crucial as we fine-tune our **GitHub Education UI** and prepare it for full release. Here's how you can get involved:

🗣️ **Share Your Thoughts:** Let us know what you think about the new design. What do you love? What could be improved? Your feedback will help us make further improvements and ensure that our platform meets your needs.** Reply to this discussion!**

💬 **Engage with Others:** Join the conversation with fellow students. Discuss your experiences, exchange ideas, and collaborate to make our platform even better.

📝**Report Bugs:** If you encounter any bugs or technical issues while using the GitHub Education UI, please report them here. Comment below if you experience issues whilst navigating the **GitHub Education UI**👇. Your reports will help us ensure a smooth transition to the new design.

We can't wait for you to explore the new UI and share your feedback with us. Together, we'll continue to create a platform that empowers learners and educators alike. 👏🎒

**Comment by pattacini** [comment]
I can't seem to find the page where to check my academic status and possibly proceed with a re-verification.

Regarding this, I think that this doc page is out of date:
- https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-teachers/apply-to-github-global-campus-as-a-teacher#applying-to-github-global-campus

**Reply by ericdrosado** [reply]
👋 Hi @pattacini ! You can check your verification status and reverify at: https://education.github.com/discount_requests/application

We'll be sure to add a link to the application page in the navigation under the user avatar as well as update our documentation. Thank you!

**Reply by pattacini** [reply]
Great! Thanks heaps @ericdrosado!

**Reply by pattacini** [reply]
I can confirm that now the link to the benefits application page is showing up in the top right corner.

<img width="378" alt="image" src="https://github.com/community/community/assets/3738070/8baa1f4d-1b71-4a82-bb0f-a76472062119">

Thanks again @ericdrosado!

**Reply by manjucmc** [reply]
It looks pretty cool. I am going to borrow some of the ideas to https://qbyte.ai

**Reply by duoduo-electronics** [reply]
你在说啥

**Reply by andyUNBOX** [reply]
The UI redesign you need is to allow students to upload documents, like PDFs of their registration status, at the verification page. Right now, the photo based system, which I have tried on phone and two computers, isn't very good. The location thing is also a joke. You do know that there are so many online courses these days? You also know how much weight you should attach to having an official email address of the institution you are claiming affiliation to?? Very frustrating experience almost not worth it to save 10 $ on Co-pilot. I'm going to try again but only one more time, ill just get another coding assistant.

**Reply by pat104471** [reply]
How could a tech company mess up a document upload tool? Using webcam to take a picture of a document that is most likely on your phone or computer is possible but annoying. How long is it going to stay like this?

**Comment by theMackabu** [comment]
Great new redesign, but on mobile, clicking sign in sends you to a 404 😅

**Reply by theMackabu** [reply]
hm not every single time though, its only happened twice out of 10 tries

**Reply by ericdrosado** [reply]
Thanks for the information @theMackabu. We'll keep an eye on this.

**Reply by elleparker** [reply]
yeah, same here....

**Comment by CASIOmax** [comment]
Pretty impressive redesign, I wasn't expecting it to be dropped soon, but overall the design is very smooth and eye catchy. I love it.

**Reply by elleparker** [reply]
agreed

**Comment by aqib-m31** [comment]
![Screenshot_2024-04-05-04-29-28-330_com.android.chrome.jpg](https://github.com/community/community/assets/63852213/22dafa4a-fb11-4d75-9c6a-e76a83f9a6d2)

Why isn't new design appearing on my devices?

**Reply by ericdrosado** [reply]
Hi @aqib-m31 !

We've looked into this issue, and you should now see the new UI after clearing your recent cache. 

Please let us know if you have any additional issues. 

Thank you for reporting!

**Reply by aqib-m31** [reply]
I can see the new UI now!
It's 😍!

**Reply by Sluckbla** [reply]
I am having the same issue, it's on only there school website github page.

**Comment by quosainhasan** [comment]
How can I turn on dark mode?

**Reply by ericdrosado** [reply]
👋 Hi @quosainhasan ! 

Unfortunately, we don't have a dark mode at this time. We'll certainly consider dark mode for a future iteration.

**Reply by bennajah** [reply]
> How can I turn on dark mode?

You can use browser extension for dark mode

**Comment by SkySingh04** [comment]
Hey there! I can't seem to find the Octernships tab on Github Educations. Can someone help me out?
![image](https://github.com/community/community/assets/114267538/b1476477-44fe-47d8-8c38-2be889cdb926)

**Reply by githubeducation** [reply]
Hi @Akash-Singh04, thanks for your interest in GitHub Octernships. Unfortunately, the program is currently on pause and we are not initiating new rounds or accepting new partners at this moment. We appreciate your understanding and support.

**Reply by ericdrosado** [reply]
👋 Hi @Akash-Singh04 - Thanks for your interest in GitHub Octernships. The program is currently on pause and we are not initiating new rounds or accepting new partners at this moment. We appreciate your understanding and support.

**Comment by riginoommen** [comment]
While trying dismissing the Notification. this throws a 403 error code - https://education.github.com/notification_dismissals?id=280

**Reply by ericdrosado** [reply]
Thanks for reporting this @riginoommen. I'll reach out to our engineering team to investigate.

**Reply by edmundmiller** [reply]
Also, getting a 502 Bad Gateway while trying to create a new assignment.

**Reply by riginoommen** [reply]
Even my assessments and update option is not working

**Reply by ericdrosado** [reply]
@riginoommen the 403 error has been fixed. Please let us know if you should see this error again. Thank you!

**Reply by riginoommen** [reply]
THis bug has fixed

**Comment by fzle** [comment]
Hey @edmundmiller you seem to be dealing with a Classroom related bug. Can you please create another discussion post with the name of the Classroom you are having trouble creating assignments in, and I'll try my best to get you sorted out.

**Comment by Brandon-Isaac** [comment]
Why is it that I finish the Communicating with markdown series and it doesnot update in my Github classroom
![Screenshot (24)](https://github.com/community/community/assets/158134226/942e3b0e-3316-402b-8125-154ac47b782d)

**Reply by ericdrosado** [reply]
Thanks for reporting @Brandon-Isaac. I'll inform our engineering team.

**Comment by Brandon-Isaac** [comment]
And why is the Github students pack taking that long for me to receive

**Comment by wpforest** [comment]
Hello,  can get a help from here ? Half of a month ago , I recevie a mail which let me reverify my education certification ，but when I follow the steps which show in the mail ---   explore the link https://education.github.com/benefits and click the "submit your information again."  But when I click "submit your information again" , it always lead me into the link ---  https://github.com/pricing  , where I can't find the place which can let me submit the information .  Is it a bug bring by this update ?

**Reply by ericdrosado** [reply]
@wpforest - You can renew your benefits at https://education.github.com/discount_requests/application

**Reply by wpforest** [reply]
Hi, Thank you , actually I have try this before, but it always lead me to   https://github.com/pricing  page . 

![price](https://github.com/community/community/assets/165918422/7bdb329b-a62e-4857-a767-e7bc793932c8)

**Comment by McAnthomRoubert** [comment]
https://docs.github.com/es/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

**Comment by riginoommen** [comment]
Currently the sync option throws the 404 error page

**Comment by malibb** [comment]
<img width="782" alt="Captura de pantalla 2024-05-16 a la(s) 1 45 11 a m" src="https://github.com/community/community/assets/16376217/4ca481d3-0a1e-4eb1-b35f-a177be60ca0d">

This doesn't work. I have the repo, but I always get a rejected message with an HTML stringified response.

{
    "html": "\u003cdiv id=\"engagement-component\" class=\"mt-7 mt-md-8\"\u003e\n  \u003csection class=\"experience__engagement\"\u003e\n    \u003ch2 class=\"h1 mb-2\" style=\"position: relative;\"\u003e\n      Take Action\n    \u003c/h2\u003e\n\n    \u003cp\u003e\n      In four easy steps, you will create a branch, commit a file to that branch, open a pull request and merge that pull request.\n    \u003c/p\u003e\n\n    \u003cdiv class=\"experience__media mt-4\"\u003e\n    \u003c/div\u003e\n\n    \u003cdiv class=\"experience__action-item text-center color-bg-default mt-7 mt-md-8\"\u003e\n      \u003ch3 class=\"mt-1\"\u003e\n        \u003cdiv class=\"h4 f6 text-normal color-fg-muted text-mono\"\u003e\n          ACTION ITEM\n        \u003c/div\u003e\n\n        Intro to GitHub Flow\n      \u003c/h3\u003e\n\n        \u003cdiv class=\"experience__actions-items-labels mt-1\"\u003e\n          \u003cspan style=\"padding-top: 2px\" data-view-component=\"true\" class=\"Label Label--danger\"\u003eRejected\u003c/span\u003e\n\n            \u003cspan style=\"padding-top: 2px\" data-view-component=\"true\" class=\"Label Label--secondary\"\u003e\n              GitHub Classroom\n\u003c/span\u003e        \u003c/div\u003e\n\n        \u003cp class=\"mt-4 f4\"\u003e\n            Learn the basics of GitHub flow in order to collaborate on projects.\n        \u003c/p\u003e\n\n        \u003cdiv class=\"experience__cta mt-4 mb-4 d-flex flex-wrap flex-justify-center\"\u003e\n            \u003ca href=\"https://classroom.github.com/a/ExYqYHCe\" data-view-component=\"true\" class=\"Button--primary Button--large Button\"\u003e  \u003cspan class=\"Button-content\"\u003e\n    \u003cspan class=\"Button-label\"\u003eStart this Experience\u003c/span\u003e\n      \u003cspan class=\"Button-visual Button-trailingVisual\"\u003e\n        \u003csvg aria-hidden=\"true\" height=\"16\" viewBox=\"0 0 16 16\" version=\"1.1\" width=\"16\" data-view-component=\"true\" class=\"octicon octicon-chevron-right\"\u003e\n    \u003cpath d=\"M6.22 3.22a.75.75 0 0 1 1.06 0l4.25 4.25a.75.75 0 0 1 0 1.06l-4.25 4.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042L9.94 8 6.22 4.28a.75.75 0 0 1 0-1.06Z\"\u003e\u003c/path\u003e\n\u003c/svg\u003e\n      \u003c/span\u003e\n  \u003c/span\u003e\n\u003c/a\u003e\n\n              \u003ca href=\"/experiences/series_intro_github_flow/verification\" data-js-engagement-component=\"true\" data-remote=\"true\" data-view-component=\"true\" class=\"Button--secondary Button--large Button\"\u003e  \u003cspan class=\"Button-content\"\u003e\n      \u003cspan class=\"Button-visual Button-leadingVisual\"\u003e\n        \u003csvg aria-hidden=\"true\" height=\"16\" viewBox=\"0 0 16 16\" version=\"1.1\" width=\"16\" data-view-component=\"true\" class=\"octicon octicon-check-circle\"\u003e\n    \u003cpath d=\"M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8Zm1.5 0a6.5 6.5 0 1 0 13 0 6.5 6.5 0 0 0-13 0Zm10.28-1.72-4.5 4.5a.75.75 0 0 1-1.06 0l-2-2a.751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018l1.47 1.47 3.97-3.97a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042Z\"\u003e\u003c/path\u003e\n\u003c/svg\u003e\n      \u003c/span\u003e\n    \u003cspan class=\"Button-label\"\u003eMark complete\u003c/span\u003e\n  \u003c/span\u003e\n\u003c/a\u003e\n              \u003cp class=\"f6 mt-3\"\u003e\n                We were unable to find a Classroom assignment for your account.\n              \u003c/p\u003e\n        \u003c/div\u003e\n    \u003c/div\u003e\n\n      \u003cdiv class=\"mt-3 pl-md-10 pr-md-10\"\u003e\n        \u003cp class=\"text-center f6 mona-sans\"\u003e\n          When you mark action items complete, you can access the next recommended Experience and make progress on any in-progress learning paths.\n        \u003c/p\u003e\n      \u003c/div\u003e\n  \u003c/section\u003e\n\n      \u003csection id=\"up-next-component\" class=\" color-shadow-medium border rounded-2 text-center p-6 m-12 mt-7 mt-md-8\"\u003e\n        \u003ch4 class=\"f6 text-normal color-fg-on-emphasis text-mono\"\u003e\n          UP NEXT\n        \u003c/h4\u003e\n\n        \u003ch2 class=\" experience__header__component f1 mb-sm-5 mb-4\"\u003e\n          Primer: Codespaces\n        \u003c/h2\u003e\n\n          \u003csvg width=\"22\" height=\"25\" aria-hidden=\"true\" viewBox=\"0 0 24 24\" version=\"1.1\" data-view-component=\"true\" class=\"octicon octicon-lock color-fg-muted mb-3\"\u003e\n    \u003cpath d=\"M6 9V7.25C6 3.845 8.503 1 12 1s6 2.845 6 6.25V9h.5a2.5 2.5 0 0 1 2.5 2.5v8a2.5 2.5 0 0 1-2.5 2.5h-13A2.5 2.5 0 0 1 3 19.5v-8A2.5 2.5 0 0 1 5.5 9Zm-1.5 2.5v8a1 1 0 0 0 1 1h13a1 1 0 0 0 1-1v-8a1 1 0 0 0-1-1h-13a1 1 0 0 0-1 1Zm3-4.25V9h9V7.25c0-2.67-1.922-4.75-4.5-4.75-2.578 0-4.5 2.08-4.5 4.75Z\"\u003e\u003c/path\u003e\n\u003c/svg\u003e\n\n          \u003cdiv class=\"experience__up-next__body color-fg-muted mx-6 px-6\"\u003e\n            \u003cp class=\"mb-3\"\u003e\n              To move on to Primer: Codespaces, \u003cspan class=\"color-fg-default text-bold\"\u003emark all your previous action items on this page complete.\u003c/span\u003e\n            \u003c/p\u003e\n            \u003cp\u003e\n              Incomplete action items will be indicated with an \u003cspan style=\"padding-top: 2px\" data-view-component=\"true\" class=\"Label Label--attention\"\u003eIncomplete\u003c/span\u003e label.\n            \u003c/p\u003e\n          \u003c/div\u003e\n      \u003c/section\u003e\n\u003c/div\u003e\n"
}

**Reply by ericdrosado** [reply]
Thanks for reporting @malibb  ! I will share this information with our engineering team.

**Comment by RIF-Creatifity** [comment]
nice

**Comment by McAnthomRoubert** [comment]
https://hipertextual.com/2024/06/recall-es-una-pesadilla-seguridad-datos-windows-11

**Comment by 2481842N** [comment]
- [ ] []()_@ _

**Comment by ariv00001** [comment]
Hey, 
I can't upload my student ID every time I push "Process my Application" the site gets updated and I have to upload it again, is there something I can do?

**Reply by ericdrosado** [reply]
Hi @ariv00001 ,

Sorry you are having difficulty uploading your student ID. Please reach out to our support team at: 

https://support.github.com/request/education

**Comment by Sewens** [comment]
I cannot upload my id card photo via webcam due to the low quality of photo.
I have the eprint of my student online report from MOE of PRC, but there's no entrance for the attachment.
While the support team said ` Individual manual reviews and status updates are no longer provided.`
So what can I do now.

**Reply by letaltom** [reply]
For me, HD wide lens webcam also produced just a blurry photo. I have tried to use phone, but switching to back camera doesn't work (3 options are provided, but all of them are front camera) and the quality from front camera is about the same as from the webcam :(.

**Comment by KavyanshKhaitan2** [comment]
Hello!
I am a student *of the organisation Delhi Public School Dibrugarh.

I am having trouble registering for Github Student.
Our organisation requires Github Student for 'some tasks' although doesnt specify what tasks.

I have already tried to register 7 times and tried to correct whatever mistake it says I have done.
Even after doing all of that, it doesnt accept my application.
![image](https://github.com/community/community/assets/73186427/fdae81ee-88c1-4732-baa7-b33ad819e6cf)
Now what do I do?

Also, this is what the support page has to say about this:
![image](https://github.com/community/community/assets/73186427/4e411b47-b484-4672-a5d9-597557c95ceb)

Any response would be greatly appreciated.

**Comment by Pog-Frog** [comment]
I love the experience that the new UI gives me , great job team 🥸

**Comment by Kunnu777** [comment]
Hey,
I can't upload my student ID every time I push "Process my Application" the site gets updated and I have to upload it again, is there something I can do? When I try to connect support it says it is a bug connect to community hub. Can't understand what to do .
![bug](https://github.com/user-attachments/assets/2fd44454-a527-4552-b606-76913a2baafb)

**Comment by h4ckthepl4net** [comment]
I would suggest to improve the page (or some parts of it) for providing a proof of academic status (can't provide a screenshot, cause student developer pack is already active for my account). I haven't seen the page for teachers, but the page for students contains IMO some real UI/UX issues. It may be hilarious or dumb, but in my case I wasn't able to find the upload button to upload photo of my documents for quite some time, I was thinking that the only option is to take a photo (and the photos taken "in app" have very bad resolution and quality). This can cause for students to be rejected because of not finding upload button and being forced to take a photo with a bad resolution. So maybe It'll be good to change the design to more understandable, especially make upload button bigger or more noticeable, also to improve taking photos from website, to get higher quality and resolution (in case of not fitting in 1 mb, maybe optimize it, resize, compress them automatically).

**Comment by bebo13133** [comment]
Hello,

I recently joined GitHub Education and I really like the design as well as the opportunity to develop and learn. However, I do have a small remark that both my colleagues and I experienced. When I was accepted into GitHub Education, I did not receive any notification that I had been admitted, and if I hadn't checked, I wouldn't have known at all! Perhaps colleagues who do not use Pro might notice the updates and their benefits on their GitHub profiles, but for those of us who were already using Pro, visually nothing has changed. It might be a good idea to send a simple template email notifying users that they have been accepted as a student or teacher into GitHub Education. Best regards and good luck!

**Comment by AdityaSrivastava185** [comment]
![github](https://github.com/user-attachments/assets/46aef6e9-4507-4e3b-8197-8d53cc53e475)

I am just thinking that github campus expert , Student developer pack and the Events can also be listed on the main page of github education as an option after `get connected`  rather than clicking on the hamburger and then finding it under the `explore more` section because it is the one of the main benefit of the github education 

Just an honest review , please do not mind if anything i said is wrong 

![github-2](https://github.com/user-attachments/assets/60784daf-d5a9-4dc9-aa8d-8dc4a3a66986)

Thank you

**Comment by evaavfx** [comment]
Unable to verify student status - camera issues. 

1. the text says i can "upload proof", but upload button does not exist. 
2. Camera is available but yields blurry pictures on preview at any distance, and application is rejected (same camera works well on other chrome based applications).  (macbook pro latest release with up to data OS and Chrome)
3. iphone 15 pro camera using chrome - only front camera is available, neither of the back cameras are yielding any visual preview nor result if button "take picture " is pressed. (system confirms that camera access has been granted each time a choice on the drop down menu is changed). With a stand, and both the laptop and phone static, front camera yields blurry picture, that results in rejection of the application. 

Any workarounds for submitting the paperwork? UX has not been tested on this flow.

**Reply by h4ckthepl4net** [reply]
The upload button exists, it is highlighted like a link in the actual "take a photo button", but I agree that there are some serious UI/UX issues in this part of platform and I have similar reply upper in this thread, so please maybe upvote it

**Reply by evaavfx** [reply]
![IMG_5618](https://github.com/user-attachments/assets/43eacedd-a247-49b7-8f0a-7e646fdd83f6)

That button definitely does not exist for me.

**Reply by AdityaSrivastava185** [reply]
This is the common problem right now , Try  to reach out the support team [here](https://github.com/orgs/community/discussions/102115#discussion-6135742)

**Reply by KavyanshKhaitan2** [reply]
What I am doing is:
1. Taking a photo on my phone
2. Sending it to my PC
3. Opening OBS
4. Adding the picture into OBS
5. Start the Virtual Camera
6. Select the Virtual Camera on the application page
7. and "Take a picture"

yeah it is very very tedious but you gotta do what you gotta do...

**Reply by MrCoolAndroid** [reply]
> What I am doing is:
> 
>     1. Taking a photo on my phone
> 
>     2. Sending it to my PC
> 
>     3. Opening OBS
> 
>     4. Adding the picture into OBS
> 
>     5. Start the Virtual Camera
> 
>     6. Select the Virtual Camera on the application page
> 
>     7. and "Take a picture"
> 
> 
> yeah it is very very tedious but you gotta do what you gotta do...

I did the same thing, only that my documents are digital, and when I take the picture the image comes out really pixelated (at least in the preview), and they rejected my application

**Comment by aqib-m31** [comment]
Local GitHub Campus Expert username on this page https://education.github.com/learner/connect ain't leading to their GitHub account. It ain't clickable.

**Comment by jobayermannan** [comment]
Loved it 👌 . waiting for the dark mode.

**Comment by romankolosok** [comment]
Sorry, my application was rejected because your multibillion-dollar company cannot add the attach local file button on the application and cannot enable the front camera to take a picture of evidence for the Student Developer Pack. Hire some web developers, Github!

**Reply by MrCoolAndroid** [reply]
Same, but I can't use the back camera, and even by using OBS (for PDF files) I can't get the image right because the quality after taking the photo sucks, and it doesn't look like they're going to do anything about it

**Reply by romankolosok** [reply]
It blurs the photo intentionally

**Comment by JorenVsyntra** [comment]
Hey everyone!

**Comment by jiayi888-cpu** [comment]
hello，I was wondering if there's a feature on github that lets me see how much time I've been using it?

**Comment by BBeast131** [comment]
Hello, developers. nice to meet you. I'm very glad to get the opportunity to meet you. I wish you success.

**Comment by server-mm404** [comment]
> **_The purpose of this discussion is to collect feedback from our verified students on our updated GitHub Education platform User Interface (UI) design._**
> 
> We're excited to roll out our redesigned **GitHub Education platform**. This overhaul marks a significant leap towards a more user-friendly, efficient, and engaging learning environment.
> 
> #### How to Access the GitHub Education UI Redesign:
> Get [verified as a learner](https://education.github.com/discount_requests/application) on GitHub Education to gain access to the new GitHub Education platform - everything you need to pursue a career in tech.
> 
> ![edu_ui_beta](https://private-user-images.githubusercontent.com/4041275/314586075-ba19202f-6a4f-4e18-8916-eca08b584583.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMDg3NjIsIm5iZiI6MTczNjMwODQ2MiwicGF0aCI6Ii80MDQxMjc1LzMxNDU4NjA3NS1iYTE5MjAyZi02YTRmLTRlMTgtODkxNi1lY2EwOGI1ODQ1ODMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMDhUMDM1NDIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2NkMDU1YmJmMDJmOTM1ZDBjZTRhOWE1YWM1YmE4NGQ4ZjFkZmFiNDc1NmVlYmNhMGMzYTY4MGVhYTI0NGQ3NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.A3PES_9IGrkqoPkxzWlkRECXRE-tJyFVE_zEElveLj8)
> 
> #### What's New:
> 🚀**Sleek Design**: Our new UI boasts a modern and sleek design, aimed at enhancing usability and aesthetics. Whether you're browsing GitHub Education resources, checking assignments, or looking to connect with your peers, you'll find the experience smoother than ever.
> 
> 🎨 **Streamlined Navigation**: We've reimagined the layout to ensure seamless navigation, making it easier for you to access essential features and resources. Say goodbye to cluttered menus! Our new side navigation groups similar topics and experiences together, making it easier than ever to find what you need.
> 
> 📣 **Enhanced Notifications**: We have also improved our platform's notifications to keep you posted on the important things that need your attention.
> 
> <img alt="Get_Connected_EDU" width="1450" src="https://private-user-images.githubusercontent.com/4041275/318470067-18105ef4-874b-4526-8ed9-0a6b5ed9c8e2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMDg3NjIsIm5iZiI6MTczNjMwODQ2MiwicGF0aCI6Ii80MDQxMjc1LzMxODQ3MDA2Ny0xODEwNWVmNC04NzRiLTQ1MjYtOGVkOS0wYTZiNWVkOWM4ZTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMDhUMDM1NDIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODE2N2JkNDU4MDJiYzU0OWY4MDFiZGFlODI3ZjRkYzAyMTg4NTkxYzgwZWMwZTk4MGQwZjc1OGM3MmJjNWZjNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.DjWXMONCvDgJmtAdLmhq3xUg1XC4pwRqq22zzRmFPzE">
> #### How You Can Help:
> Your feedback is crucial as we fine-tune our **GitHub Education UI** and prepare it for full release. Here's how you can get involved:
> 
> 🗣️ **Share Your Thoughts:** Let us know what you think about the new design. What do you love? What could be improved? Your feedback will help us make further improvements and ensure that our platform meets your needs.** Reply to this discussion!**
> 
> 💬 **Engage with Others:** Join the conversation with fellow students. Discuss your experiences, exchange ideas, and collaborate to make our platform even better.
> 
> 📝**Report Bugs:** If you encounter any bugs or technical issues while using the GitHub Education UI, please report them here. Comment below if you experience issues whilst navigating the **GitHub Education UI**👇. Your reports will help us ensure a smooth transition to the new design.
> 
> We can't wait for you to explore the new UI and share your feedback with us. Together, we'll continue to create a platform that empowers learners and educators alike. 👏🎒

**Comment by ApplebaumIan** [comment]
Is there a way to not be forced to use the Fork feature in GitHub Classroom? The old way was much better for my projects course as I used GitHub Classroom to form project teams and disseminate project templates. Now every time students make a PR, they’re confused because it defaults to the parent repository and not their projects. This is becoming a nuisance and really has me considering searching for alternatives.

**Comment by Wid1ey** [comment]
![image](https://github.com/user-attachments/assets/b6a3be54-72df-4a97-ac60-3cb1622eff98)

**Comment by Waiwaitiyyt** [comment]
Dark mode pls
